tvm/cli.py

```diff
@@ -137,11 +139,45 @@ def setup_version_virtualenv(version=None) -> None:
 
 
 @click.command(name="list")
-@click.option('-l', '--limit', default=10, help='number of `latest versions` to list')
+@click.option("-l", "--limit", default=10, help="number of `latest versions` to list")
 def list_versions(limit: int):
     """
     Get all the versions from github.
 
+    Print and mark the both the installed ones and the current.
+    """
+    repository = VersionManagerGitRepository()
+    lister = TutorVersionLister(repository=repository)
+    version_names = lister()
+    version_names = sorted(version_names, reverse=True, key=LooseVersion)
+    local_versions = repository.local_versions(TVM_PATH)
+    global_active = repository.current_version(TVM_PATH)
+    project_version = None
```

Comment:
Please use the old method to get the project version. In the future, we can replace it when we migrate it.